build.py

```diff
@@ -43,7 +44,7 @@
     {'name': 'pg-dev', 'tag': '11', 'args': {'PG_VERSION': '11'}},
     {'name': 'pg-dev', 'tag': '12', 'args': {'PG_VERSION': '12'}},
     {'name': 'mongo-dev', 'tag': '4.1', 'args': {'BASE_TAG': '4.1'}},
-    {'name': 'redis-dev', 'tag': '6'}
+    {'name': 'redis-dev', 'tag': '6', 'path': 'redis'}
```

Comment:
I moved the folder in the correct location instead. It doesn't need the path specified anymore.